sdk/communication/Azure.Communication.Calling.Server/samples/Sample1_CreateCallAsync.md

```diff
@@ -0,0 +1,41 @@
+# Create Call
+
+This sample demonstrates how to make a call to a recipient phone number.
+
+To get started you'll need a Communication Service Resource.  See [README][README] for prerequisites and instructions.
+
+## Creating an `ServerCallingClient`
+
+Server Calling clients can be authenticated using the connection string acquired from an Azure Communication Resource in the Azure Portal.
+
+```C# Snippet:Azure_Communication_ServerCalling_Tests_Samples_CreateServerCallingClient
+var connectionString = "<connection_string>"; // Find your Communication Services resource in the Azure portal
+CallClient client = new CallClient(connectionString);
+```
+
+## Make a call to a phone number recipient
+
+To make a Call, call the `CreateCall` or `CreateCallAsync` function from the `CallClient`. The returned value is `CreateCallResponse` objects that contains the created Call's Id if succeed, else throws a RequestFailedException.
+```C# Snippet:Azure_Communication_Call_Tests_CreateCallOptions
+var createCallOption = new CreateCallOptions(
+       new Uri(TestEnvironment.AppCallbackUrl),
+       new List<CallModality> { CallModality.Audio },
+       new List<EventSubscriptionType>
+       {
+           EventSubscriptionType.ParticipantsUpdated,
+           EventSubscriptionType.DtmfReceived
+       });
+```
+```C# Snippet:Azure_Communication_Call_Tests_CreateCallAsync
```

Comment:
The syntax for this snippet seems wrong... it is missing the end `)` bracket that closes off `CreateCallAsync`

Reply:
Thanks for catching this. Addressed.